src/PIL/Image.py

```diff
@@ -633,19 +633,34 @@ def _repr_pretty_(self, p, cycle):
             )
         )
 
-    def _repr_png_(self):
+    def _repr_image(self, image_format):
         """iPython display hook support
 
-        :returns: png version of the image as bytes
+        :param image_format: Image format.
+        :returns: image as bytes, saved into the given format.
         """
         b = io.BytesIO()
         try:
-            self.save(b, "PNG")
+            self.save(b, image_format)
         except Exception as e:
-            msg = "Could not save to PNG for display"
+            msg = f"Could not save to {image_format} for display"
             raise ValueError(msg) from e
         return b.getvalue()
 
+    def _repr_png_(self):
+        """iPython display hook support for PNG format.
+
+        :returns: png version of the image as bytes
+        """
+        return self._repr_image("PNG")
+
+    def _repr_jpeg_(self):
+        """iPython display hook support for JPEG format.
+
+        :returns: jpg version of the image as bytes
```

Comment:
```suggestion
        :returns: jpeg version of the image as bytes
```
Just for consistency with the surrounding text.